db/migrate/20230328200147_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :posts_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] This table has duplicate that needs to be removed before migration could be run

### Duplicate table
![create-user2](https://user-images.githubusercontent.com/26597432/230608441-eca24ee3-6cda-4813-99ec-4ff9034f20c2.png)


### Error while trying to run migrate
![duplicateMigrate-error](https://user-images.githubusercontent.com/26597432/230608405-3ec2ed4d-c767-492c-8000-0deeabe36aaa.png)